Oglle/dist/presence.js

```diff
@@ -10,7 +10,10 @@ presence.on("UpdateData", () => {
     };
     presenceData.startTimestamp = browsingStamp;
 
-    if(window.location.pathname.includes("")) {
+    if(window.location.pathname.includes()) {
+        presenceData.details = "Inicio"
+    }

```

Comment:
Can just be removed since it doesnt do anything.

Reply:
If removed, the home page will not be shown in the user's status with our presence, since the home page does not need to have "/home" or in this case in Portuguese "/inicio"!

Reply:
![image](https://user-images.githubusercontent.com/34792838/74587875-34877e00-4ff8-11ea-957f-cb54c9c69405.png)
This one should fix that tho.. Try it.

Reply:
I already tested it, but sometimes there is no "/" in the url, then it does not recognize it as the beginning, in this case if it does not have the "/" it will be the beginning, but if it does, it will includes("/") when the url has "/" it will not appear as a start, and if it leaves only includes() when it has "/" it will not appear as a start either...